jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  testEnvironment: 'jsdom',
```

Comment:
Had to add this after upgrading `jest`: https://jestjs.io/docs/configuration#testenvironment-string

> If you are building a web app, you can use a browser-like environment through [jsdom](https://github.com/jsdom/jsdom) instead.